core/main/src/bootstrap/logging_bootstrap_step.rs

```diff
@@ -0,0 +1,44 @@
+use crate::{state::bootstrap_state::BootstrapState, SEMVER_LIGHTWEIGHT};
+use ripple_sdk::{
+    async_trait::async_trait,
+    framework::{bootstrap::Bootstep, RippleResponse},
+    log,
+    utils::logger::init_and_configure_logger,
+};
+
+pub struct LoggingBootstrapStep;
+
+#[async_trait]
+impl Bootstep<BootstrapState> for LoggingBootstrapStep {
+    fn get_name(&self) -> String {
+        "LoggingBootstrapStep".into()
+    }
+
+    async fn setup(&self, state: BootstrapState) -> RippleResponse {
+        //let device_manifest = &state.platform_state.device_manifest.configuration;
+        let manifest = state.platform_state.get_device_manifest();
+        let log_signal_level = match manifest
```

Comment:
Take a look at the Create Feature Flags of the log crate, there's a `serde` feature that allows the de/serialization of `LevelFilter`. If adding that feature doesn't add too much overhead, all of this can be removed and your entries in the manifest would just be `Info`, `Debug`, `Off`, etc.